installation/sql/postgresql/base.sql

```diff
@@ -365,6 +365,7 @@ INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder",
 (0, 'plg_task_checkfiles', 'plugin', 'checkfiles', 'task', 0, 1, 1, 0, 1, '', '{}', '', 1, 0),
 (0, 'plg_task_requests', 'plugin', 'requests', 'task', 0, 1, 1, 0, 1, '', '{}', '', 3, 0),
 (0, 'plg_task_sitestatus', 'plugin', 'sitestatus', 'task', 0, 1, 1, 0, 1, '', '{}', '', 4, 0),
+(0, 'plg_task_globalcheckin', 'plugin', 'checkin', 'task', 0, 1, 1, 0, 0, '', '{}', '', 2, 0),
```

Comment:
```suggestion
(0, 'plg_task_checkfiles', 'plugin', 'checkfiles', 'task', 0, 1, 1, 0, 1, '', '{}', '', 1, 0),
(0, 'plg_task_globalcheckin', 'plugin', 'checkin', 'task', 0, 1, 1, 0, 0, '', '{}', '', 2, 0),
(0, 'plg_task_requests', 'plugin', 'requests', 'task', 0, 1, 1, 0, 1, '', '{}', '', 3, 0),
(0, 'plg_task_sitestatus', 'plugin', 'sitestatus', 'task', 0, 1, 1, 0, 1, '', '{}', '', 4, 0),
```
Alpha-order (the order number is already 2 as it should be)

Reply:
Other question: Shall it really be named "plg_task_globalcheckin"? If so, file names and language string key and so on have to be adjusted to that. Or change the name back to like it was before the last commit, "plg_task_checkin".

Reply:
name will be changed as requested by Tobias

Reply:
Then you have to rename files and folders and language keys.